editoast/src/views/train_schedule.rs

```diff
@@ -787,6 +795,58 @@ async fn project_path(
     Ok(Json(project_path_result))
 }
 
+#[derive(Debug, Default, Clone, Serialize, Deserialize, ToSchema)]
+struct TrackOccupancyForm {
+    train_schedule_ids: Vec<i64>,
+    op_id: i64,
+    infra_id: i64,
+    electrical_profile_set_id: Option<i64>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+struct TrackOccupancy {
+    time_begin: DateTime<Utc>,
+    #[schema(value_type = chrono::Duration, example = "PT5M")]
```

Comment:
It could be interesting to implement ToSchema manually for PositiveDurantion and provide the example there. This way you can just inline it here and forget about how it's represented.

Reply:
Good idea. I will do it in another PR. :)

Reply:
Sure, thanks! It can be an enhancement good first issue